### PR TITLE
added to Newsfeed-Components

### PR DESCRIPTION
**Pull request by marianahuh**
Added to Newsfeed-Components

### CODE REVIEW

**Comment by marianahuh** [comment]
Submitting MVP for Newsfeed-Components.

File: Article/Article.js [hunk 1]
```diff
@@ -1,29 +1,36 @@
...
+    this.articleClose = article.querySelector('.close');
+    // create a reference to the ".expandButton" class.
+    this.expandButton = article.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = 'Expand';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => this.expandArticle());
```
Comment (ielvisd) [inline Article/Article.js:15]: Good use of an arrow function to target your `expandArticle` method.

File: Menu/Menu.js [hunk 2]
```diff
@@ -1,10 +1,14 @@
-
-const toggleMenu = () => {
-  // Toggle the "menu--open" class on your menu refence. 
-}
-
 // Start Here: Create a reference to the ".menu" class
-const menu;
+const menu = document.querySelector('.menu');
```
Comment (ielvisd) [inline Menu/Menu.js:2]: Good use of querySelector to target your menu and menu-button classes.

**Review by ielvisd — APPROVED** [review]
Nice work Mariana, functionality works and your code looks clean and concise.

**Pull request merged**